basics/keeping-up-to-date/upgrade.md

```diff
@@ -28,16 +28,15 @@ This guide gives you the full control on the process. This one has been applied
 
 ### Release download
 
-The first step is to download the latest version on https://www.prestashop.com/.
+The first step is to download the latest version on https://github.com/PrestaShop/PrestaShop/releases.
 
-If you upgrade to another version of PrestaShop 1.6, the release files
-can be found in a [dedicated archives list](https://www.prestashop.com/en/previous-versions?version=1.6).
+If you upgrade to another version of PrestaShop 1.6, the release files can be found in a [dedicated archives list](https://www.prestashop.com/en/previous-versions?version=1.6).
```

Comment:
Why not the PrestaShop-1.6 repo as you mention the PrestaShop one for latest release ? It will remove the link to prestashop.com, isn't it ?

Reply:
Because we don't have any list (except on prestashop.com) where we have only 1.6 versions

Reply:
Oh. There is just tags and no release on this repository. Okay' ! :)